README.md

```diff
@@ -1,14 +1,17 @@
-# Knative Sample Controller
+# Scaling-Group
+[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://knative.slack.com)
```

Comment:
Just as an FYI, we're going to be moving to the CNCF Slack in the near future, so this'll need an update once that happens.